website/moneybirdsynchronization/services.py

```diff
@@ -0,0 +1,246 @@
+        instance.payment.save()
+    except Administration.Error as e:
+        emails.send_sync_error(e, instance.payment)
+
+
+def delete_payment(instance):
+    if settings.MONEYBIRD_SYNC_ENABLED is False:
+        return
+
+    apiservice = get_moneybird_api_service()
+    apiservice.delete_external_invoice(instance)
+
+
+def sync_contacts():
+    if settings.MONEYBIRD_SYNC_ENABLED is False:
+        return
+
+    apiservice = get_moneybird_api_service()
+
+    members_without_contact = Member.objects.filter(
+        ~Q(
+            id__in=Subquery(
+                MoneybirdContact.objects.filter(member=OuterRef("pk")).values("member")
+            )
+        )
+    )
+
+    for member in members_without_contact:
+        contact = MoneybirdContact(member=member)
+        contact.save()
+
+    # fetch contact ids from moneybird
+    api_response = apiservice.api.get("contacts")
+
+    # fetch contact ids from contact model
+    contact_info = [
+        contact.get_moneybird_info() for contact in MoneybirdContact.objects.all()
+    ]
+
+    # find contacts in contact model that are not in moneybird and add to moneybird
+    moneybird_ids = [int(info["id"]) for info in api_response]
+    for contact in contact_info:
+        if contact["id"] is None or int(contact["id"]) not in moneybird_ids:
+            contact = MoneybirdContact.objects.get(
+                member=Member.objects.get(pk=contact["pk"])
+            )
+            response = apiservice.add_contact_to_moneybird(contact)
+            contact.moneybird_id = response["id"]
+            contact.moneybird_version = response["version"]
+            contact.save()
+
+    moneybird_info = []
+    for contact in api_response:
+        if len(contact["custom_fields"]) > 0:
+            moneybird_info.append(
+                {
+                    "id": contact["id"],
+                    "version": contact["version"],
+                    "pk": contact["custom_fields"][0]["value"],
+                }
+            )
+
+    ids = [info["id"] for info in contact_info]
+    for moneybird in moneybird_info:
+        if int(moneybird["id"]) not in ids:
+            apiservice.delete_contact(moneybird["id"])
+
+
+def sync_statements():
+    if settings.MONEYBIRD_SYNC_ENABLED is False:
+        return
+
+    apiservice = get_moneybird_api_service()
+
+    date = datetime.date.today()
+    new_card_payments = Payment.objects.filter(
+        type=Payment.CARD,
+        created_at__year=date.year,
+        created_at__month=date.month,
+        created_at__day=date.day,
+        moneybird_financial_statement_id=None,
+    )
+    new_cash_payments = Payment.objects.filter(
+        type=Payment.CASH,
+        created_at__year=date.year,
+        created_at__month=date.month,
+        created_at__day=date.day,
+        moneybird_financial_statement_id=None,
+    )
+
+    card_account_id = apiservice.get_financial_account_id(
+        settings.PAYMENT_TYPE_TO_FINANCIAL_ACCOUNT_MAPPING["card"]
```

Comment:
here you can just use `Payment.CARD` I think

Reply:
no since the value of `Payment.CARD` is different from the current value that is passed